src/components/keyboardnavigation.js

```diff
@@ -126,6 +126,41 @@ define(["inputManager", "layoutManager"], function (inputManager, layoutManager)
         });
     }
 
+    function isGamepadConnected() {
+        var gamepads = navigator.getGamepads();
+        var i;
+        var len;
+        for (i = 0, len = gamepads.length; i < len; i++) {
+            var gamepad = gamepads[i];
+            if (gamepad) {
+                console.log(gamepad);
+                return true;
+            }
+        }
+        return false;
+    }
+
+    function attachGamepad(e) {
+        if (isGamepadConnected()) {
+            require(["components/input/gamepadtokey"]);
+            console.log("Gamepad connected! Attaching gamepadtokey.js script");
+        }
+    }
+
+    function dettachGamepad(e) {
+        if (!isGamepadConnected()) {
+            delete require.cache[require(["components/input/gamepadtokey"])];
+            console.log("Gamepad disconnected! No other gamepads are connected, dettaching gamepadtokey.js");
+        } else {
+            console.log("Gamepad disconnected! There are gamepads still connected.");
+        }
+    }
+
+    // No need to check for gamepads manually at load time, the eventhandler will be fired for that
+    window.addEventListener("gamepaddisconnected", dettachGamepad);
+    window.addEventListener("gamepadconnected", attachGamepad);
+    require(["components/input/mouseManager"]);
```

Comment:
Since `mouseManager` is on its own, maybe this line should be moved to `site.js`.
https://github.com/jellyfin/jellyfin-web/blob/bf4207cc3657b28498bdabc8ee7052ab0ada6c2b/src/scripts/site.js#L485-L491

Reply:
Hmm, for me no problem, but maybe makes more sense to have all the input controllers in one place?

Reply:
Btw by seeing this... Maybe we can invoke ``gamepadtokey`` just like ``keyboardnavigation`` and ``autofocuser`` are invoked? So, whenever we want to kill it, we can call ``disable()``

Pretty noob with JavaScript packages as you can see, so probably this makes no sense, so any feedback about this is appreciated! :)

Reply:
The one place in this case would be `site.js` and the input folder rather than the `keyboardnavigation.js` file.

Reply:
> Pretty noob with JavaScript packages as you can see

I am not better :stuck_out_tongue: 

> Maybe we can invoke `gamepadtokey` just like `keyboardnavigation` and `autofocuser` are invoked? So, whenever we want to kill it, we can call `disable()`

Actually, these two can be used without dedicated `enable`. I think this makes sense in case of delayed activation and/or external control. Won't do worse, but gamepad does not need this.